cli/pkg/update/update.go

```diff
@@ -64,12 +65,12 @@ func LatestVersion(ctx context.Context) (string, error) {
 			return "", err
 		}
 
-		updatedAt, err := time.Parse("2006-01-02 15:04", cachedVersionUpdatedAt)
+		updatedAt, err := time.Parse(cmdutil.TSFormatLayout, cachedVersionUpdatedAt)
 		if err != nil {
 			return "", err
 		}
 
-		if time.Since(updatedAt).Hours() < versionCheckTTL.Hours() {
+		if time.Since(updatedAt.UTC()).Hours() < versionCheckTTL.Hours() {
```

Comment:
1. `UTC` will not be needed here when using `time.RFC3339`
2. Don't do `Hours()`, just compare the time deltas